app/src/main/java/com/infomaniak/mail/ui/main/folder/ThreadListFragment.kt

```diff
@@ -182,9 +181,10 @@ class ThreadListFragment : Fragment(), SwipeRefreshLayout.OnRefreshListener {
 
         threadsList.swipeListener = object : OnItemSwipeListener<Any> {
             override fun onItemSwiped(position: Int, direction: SwipeDirection, item: Any): Boolean {
+                item as Thread
```

Comment:
Why here ?

Reply:
Because we'll also need it there:
```
SwipeDirection.RIGHT_TO_LEFT -> notYetImplemented() // TODO: Delete thread
```

We could wait for the day that we implement the `RIGHT_TO_LEFT` swipe, but @JorisBodin asked to do it now.